docker/hq-compose.yml

```diff
@@ -95,8 +95,8 @@ services:
           KAFKA_BROKER_ID: 1
           CUSTOM_INIT_SCRIPT: "curl https://raw.githubusercontent.com/vishnubob/wait-for-it/c096cface5fbd9f2d6b037391dfecae6fde1362e/wait-for-it.sh > wait.sh && chmod +x ./wait.sh && ./wait.sh zookeeper:2181 || exit 1"
           KAFKA_PORT: 9092
-          KAFKA_LISTENERS: PLAINTEXT://:9092
-          KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092
+          LISTENERS: PLAINTEXT://{host IP address}:9092
```

Comment:
I don't think this change is correct - for docker the previous settings were correct.

Reply:
I tested it on local linux system it worked , so wanted to test it on Travis as well

Reply:
As you can see from the [docker image docs](https://github.com/wurstmeister/kafka-docker#listener-configuration) the docker environment variables are named differently:

###Example
The example environment below:
```
KAFKA_ADVERTISED_LISTENERS: INSIDE://:9092,OUTSIDE://_{HOSTNAME_COMMAND}:9094
KAFKA_LISTENERS: INSIDE://:9092,OUTSIDE://:9094
```
Will result in the following broker config:
```
advertised.listeners = OUTSIDE://ec2-xx-xx-xxx-xx.us-west-2.compute.amazonaws.com:9094,INSIDE://:9092
listeners = OUTSIDE://:9094,INSIDE://:9092
inter.broker.listener.name = INSIDE
```

Reply:
Strange both changes are working , which one shall we keep ?

Reply:
it only works because it's using default values. I tested the defaults locally and they appear to work so I think we should leave these docker settings out completely.